creator-node/sequelize/migrations/20200911004845-allow-track-and-audiusUsers-appends.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+      BEGIN;
+      -- replace a table in place with extra trackBlockchainId column
+      CREATE TABLE "Files_new" ( like "Files" );
+      ALTER TABLE "Files_new" ADD COLUMN "trackBlockchainId" INTEGER;
+      INSERT INTO "Files_new" (SELECT f.*, t."blockchainId" FROM "Files" f LEFT OUTER JOIN "Tracks" t ON f."trackUUID" = t."trackUUID");
```

Comment:
just want to confirm behavior here for files that do not have a trackUUId - would t.blockchainId just be null?
also, how does it know to match the `t."blockchainId"` value to the `"trackBlockchainId"` field? is it just by ordering?
also to confirm correctness - can we select all columns from Files.old then select all columns except for new one and confirm data is exactly equal? just thinking about best way to confirm success

Reply:
yes it's correct that blockchainId would just be null. As far as how it knows to match f.* passes in all columns from the other file table. the order of the columns is actually a postgres property that's stored and it inserts it in order of attributes ASC and since trackBlockchainId was added last, it's got the highest number so it goes at the end.

as for correctness of this if you'd like to be 100% certain it's right we can add the list of columns to put in a deterministic order. i can do this too, just to be safe

Reply:
just bumping - we should handle this after merge

Reply:
this should be fixed. the properties are explicitly named now